src/server/bullet.test.js

```diff
@@ -0,0 +1,42 @@
+const Bullet = require('./bullet');
+const Constants = require('../shared/constants');
+
+describe('Bullet', () => {
+  const directionUp = 0;
+  const directionDown = Math.PI;
+
+  describe('update', () => {
+    it('should move at bullet speed', () => {
+      const parentID = '123';
```

Comment:
none of these 3 cases use `parentID` so let's just inline this value to save space and make it something informational, e.g.

```js
const bullet = new Bullet('test-parent-id', x, y, dir);
```